Resources/views/CRUD/edit_orm_one_to_many_inline_tabs.html.twig

```diff
@@ -35,18 +35,20 @@ file that was distributed with this source code.
                             <fieldset>
                                 <div class="sonata-ba-collapsed-fields">
                                     {% for field_name in form_group.fields %}
-                                        {% set nested_field = nested_group_field.children[field_name] %}
-                                        <div class="sonata-ba-field-{{ id }}-{{ field_name }}">
-                                            {% if associationAdmin.formfielddescriptions[field_name] is defined %}
-                                                {{ form_row(nested_field, {
-                                                    'inline': 'natural',
-                                                    'edit'  : 'inline'
-                                                }) }}
-                                                {% set dummy = nested_group_field.setrendered %}
-                                            {% else %}
-                                                {{ form_row(nested_field) }}
-                                            {% endif %}
-                                        </div>
+                                        {% if nested_group_field.children[field_name] is defined %}
```

Comment:
I think you can combine the `for` and the `if`, like this : http://twig.sensiolabs.org/doc/2.x/tags/for.html#adding-a-condition